sklearn/model_selection/_split.py

```diff
@@ -84,6 +84,12 @@ def split(self, X, y=None, groups=None):
 
         test : ndarray
             The testing set indices for that split.
+
+        Note
+        ----
+        Multiple calls to the ``split`` method will not return identical
+        training or testing sets if ``random_state`` parameter exists and is
```

Comment:
Sorry for coming to this very late. What do you suggest as the right thing to do?

```py
"This *may not* be deterministic if `random_state`, if available, is not explicitly set while initializing the class"
```

Sounds okay to you?